sdk/storage/AzureStorageBlob/Source/StorageBlobClientOptions.swift

```diff
@@ -0,0 +1,283 @@
+// --------------------------------------------------------------------------
```

Comment:
It looks like this got duplicated? The existing copy is at `Source/DataStructures/StorageBlobClientOptions.swift`.

Reply:
Removed.